feature-libs/order/assets/translations/translations.ts

```diff
@@ -20,5 +20,9 @@ export const orderTranslationChunksConfig: TranslationChunksConfig = {
     'returnRequest',
     'reorder',
   ],
-  orderEnhancedUI: ['orderHistoryEnhancedUI', 'orderDetailsEnhancedUI'],
+  orderEnhancedUI: [
+    'orderHistoryEnhancedUI',
+    'orderDetailsEnhancedUI',
+    'myAccountView',
+  ],
```

Comment:
I think we should align with @anjana-bl on the consistent convention of naming the namespaces of translations.

Currently: `orderEnhancedUI.json` will be lazy loaded whenever on the page appears a key starting with `orderHistoryEnhancedUI`, `orderDetailsEnhancedUI` or `myAccountView`.

I guess maybe we'd like to rename the namespace from `myAccountView` to `orderMyAccountView`?

Reply:
Hi Kris, I followed the finalized naming convention - (MyAccountV2)(feature)